server/routes/viewingLicences/handlers/viewCases.ts

```diff
@@ -38,7 +39,10 @@ export default class ViewAndPrintCaseRoutes {
             _.head(c.licences).status !== LicenceStatus.NOT_IN_PILOT &&
             _.head(c.licences).status !== LicenceStatus.OOS_RECALL &&
             _.head(c.licences).status !== LicenceStatus.OOS_BOTUS &&
-            _.head(c.licences).status !== LicenceStatus.IN_PROGRESS,
+            _.head(c.licences).status !== LicenceStatus.IN_PROGRESS &&
+            _.head(c.licences).status !== LicenceStatus.VARIATION_IN_PROGRESS &&
+            _.head(c.licences).status !== LicenceStatus.VARIATION_APPROVED &&
+            _.head(c.licences).status !== LicenceStatus.VARIATION_SUBMITTED,
```

Comment:
the variation types should not be clickable as OMU (prison) staff can't do variations